security/libu2f-server/Portfile

```diff
@@ -23,16 +23,17 @@ master_sites        https://developers.yubico.com/${name}/Releases/
 
 use_xz              yes
 
-checksums           rmd160  fe8dffe7f62377c0f409dee112adb73cf8c1a928 \
-                    sha256  a618f59051209d6d70c24cf42d64c9b67bd7dd5946b6dbd2c649181d7e8f1f6e
+checksums           rmd160  cbd67ad4cc3ab73362749576e9debd426a2de250 \
+                    sha256  8dcd3caeacebef6e36a42462039fd035e45fa85653dcb2013f45e15aad49a277 \
+                    size    269128
 
 depends_build       port:check \
-                    port:pkgconfig
+                    port:pkgconfig \
+                    port:gengetopt
+                    
```

Comment:
Trailing whitespace.

Reply:
Cool! Builds for me! 

If you don't mind, I will let you take it from here though, if you want. Just trying to help keep you up to date :>